app/controllers/home_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
     @rentals = if @current_user.groups.where(name: 'admin').present?
                  Rental.all
                else
-                 @current_user.rentals
+                 Rental.where(renter: @current_user)
```

Comment:
Now that I understand why, I'd recommend a scope so that you could say something like

```ruby
Rental.rented_by @current_user
```